packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -74,6 +76,13 @@ class DebuggerController extends DisposableController
 
   ValueListenable<String> get exceptionPauseMode => _exceptionPauseMode;
 
+  final _stdio = ValueNotifier<String>('');
```

Comment:
making stdio and stdout be a List<String> would let views displaying this data in the case of a very large input be more efficient as the ListView would be able to display only the rows in view rather than having to first recompute line breaks.

Reply:
Hmm, I'm guessing I could just use lines of `Text` in the `ListView`. This might also make it easier to start using RichTexts in order to differentiate between stdout and stderr.

Reply:
I'll implement this and follow up on the PR.

Reply:
The other thing to consider is whether you want to jump straight to using our TreeTable for the console output here. That would be nice for in the future displaying collapsible rows. The TreeTable also needs to support scroll to bottom functionality.

Reply:
I'd like to just bite off the ListView work for this PR -